.github/workflows/test.yml

```diff
@@ -107,6 +107,9 @@ jobs:
           RUST_BACKTRACE: full
         run: cargo test --verbose -p pcb -- --ignored
 
+      - name: Build examples
```

Comment:
Should we build everything in the examples folder?

Reply:
yes definitely, done